packages/web/src/components/nav/desktop/PlaylistLibrary.tsx

```diff
@@ -239,6 +241,110 @@ const PlaylistFolderNavItem = ({
   )
 }
 
+type PlaylistNavItemProps = {
+  playlist: AccountCollection
+  url: string
+  addTrack: (trackId: ID) => void
+  isOwner: boolean
+  onReorder: (
+    draggingId: ID | SmartCollectionVariant,
+    droppingId: ID | SmartCollectionVariant
+  ) => void
+  hasUpdate?: boolean
+  dragging: boolean
+  draggingKind: string
+  onClickPlaylist: (id: ID, hasUpdate: boolean) => void
+  onClickEdit?: (id: ID) => void
+}
+const PlaylistNavItem = ({
+  playlist,
+  hasUpdate = false,
+  url,
+  addTrack,
+  isOwner,
+  onReorder,
+  dragging,
+  draggingKind,
+  onClickPlaylist,
+  onClickEdit
+}: PlaylistNavItemProps) => {
+  const { id, name } = playlist
+  const [isHovering, setIsHovering] = useState(false)
+
+  return (
+    <Droppable
+      key={id}
+      className={navColumnStyles.droppable}
+      hoverClassName={navColumnStyles.droppableHover}
+      onDrop={addTrack}
+      acceptedKinds={['track']}
```

Comment:
Should this be 'playlist'?

Reply:
this means it accepts tracks (it being a playlist)